src/software/visualizer/widgets/main_widget.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include "software/visualizer/geom/geometry_conversion.h"
 
-MainWidget::MainWidget(QWidget* parent)
+MainWidget::MainWidget(std::shared_ptr<ThunderbotsConfig> config, QWidget* parent)
```

Comment:
Just a question here, why are we passing the config here if it is not assigned to a member variable?

Reply:
Looks like it's passed into `setupAIControls` on line 36?

Reply:
Yes you are right, didn't see that